packages/service/src/utils/db.test.ts

```diff
@@ -0,0 +1,31 @@
+import db from "./db";
+
```

Comment:
there are one to many files named `db.test.ts` here, lets combine them or name them differently.